### DISCUSSION TITLE
IOS App crash | Won't open anymore

### DISCUSSION THREAD

**Discussion by lodembeep** [community/community#110120] [Mobile] [answered]
### Select Topic Area

Bug

### Body

I don't know exactly what happened but since a few days now I am unable to open the app. It crashed, and ever since i get a black screen for a second then back to the home screen.

It is a typical start-up crash caused by some corrupt or error data saved in my device.

The problem is that I use the Github app as 2FA and to get notifications. I will try to use another 2FA method, but until this is fixed, I won't be using any other Github product.

### Update
Seems like they are working on a fix here : [Testflight confirmed v1.150 bug](https://github.com/orgs/community/discussions/109554#discussioncomment-8567425)

Fixed in 1.151.0 on TestFlight

**Comment by jamesmcaleer** [comment]
Not an answer but just letting you know this happened to me as well. Deleting and reinstalling did not work.

**Reply by AydinGokce** [reply]
I am experiencing the same issue. Locked out of anything that requires 2FA :/

**Reply by HenSquared** [reply]
@jamesmcaleer , @AydinGokce 

Sorry for the inconvenience. Can you also share the OS version installed on your device? We are actively investigating this issue.

**Reply by Daniel201299** [reply]
i am facing the same issue. Device iphone 13 pro with ios 17.3.1 installed.

**Comment by sbslee** [comment]
Same thing for me—I'm a user in South Korea. I attempted to turn off my iPhone XR (iOS 16.3.1) and reboot, but to no avail. Following that, I tried reinstalling the latest version (1.150.0), but it still isn't working. I hope this gets fixed soon.

**Reply by HenSquared** [reply]
Thank you for the information, we're actively investigating this issue. We will reply here as soon as we have updates. Sorry for the inconvenience.

**Comment by amosyou** [comment]
encountering the same bug as well. I was opening a github link from twitter and the github app opened for a short second before crashing, and now I can't open it anymore (even uninstalling / restarting the phone doesn't work)

**Reply by HenSquared** [reply]
Thanks for the report @amosyou. What OS version is installed on your device?

**Reply by amosyou** [reply]
iOS 16.3.1 (iPhone 11 Pro Max)

**Reply by HenSquared** [reply]
Thank you :)

**Comment by QuinnBast** [comment]
I’ve been having this issue for the last few days also. Uninstall and reinstall did not work. Kinda rough because I can’t login with 2FA with the app crashing, I have to manually use an Authenticator app

**Reply by HenSquared** [reply]
Hi @QuinnBast , can you share your OS version you are running? 

Thanks and sorry for the inconvenience here.

**Comment by beverm2391** [comment]
same issue here

**Reply by HenSquared** [reply]
@beverm2391 , what OS version are you running on your device? 

Thanks for the report and any additional information. We are actively investigating this. Sorry for the inconvenience.

**Comment by matthewfurr** [comment]
Running into the same issue as well. I’m on an iPhone 12 running iOS 16.1.2. I have has this issue since Feb 23rd.

**Reply by HenSquared** [reply]
@matthewfurr , thanks for the OS version, this is helpful in our investigation. Sorry for the inconvenience. We'll reply here once we have updates.

**Comment by HenSquared** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> I don't know exactly what happened but since a few days now I am unable to open the app. It crashed, and ever since i get a black screen for a second then back to the home screen.
> 
> It is a typical start-up crash caused by some corrupt or error data saved in my device.
> 
> The problem is that I use the Github app as 2FA and to get notifications. I will try to use another 2FA method, but until this is fixed, I won't be using any other Github product.

We're actively investigating this issue. May you share your OS version? This will help us in our investigation. Sorry for the inconvenience and thanks for the report and any additional information.

**Reply by lodembeep** [reply]
Iphone 8 plus IOS 16.1.1

**Reply by HenSquared** [reply]
Thanks @lodembeep

**Comment by ghost** [comment]
Same here. IPhone 14 iOS 16.2

**Reply by ghost** [reply]
https://github.com/community/community/assets/74627744/6eacddb8-7c7c-42b5-933b-7f9a1e9504f9

**Reply by HenSquared** [reply]
thanks @mooseyacky

**Comment by hongdoojung** [comment]
same here. I'm using IPhone 12 & iOS 17.3.1.

**Comment by lodembeep** [comment, marked as answer]
[The fiix is ont its way](https://github.com/orgs/community/discussions/109554#discussioncomment-8593248)
